src/core/surface/channel.c

```diff
@@ -179,6 +180,16 @@ static grpc_call *grpc_channel_create_call_internal(
     send_metadata[num_metadata++] = GRPC_MDELEM_REF(channel->default_authority);
   }
 
+  if (propagation_mask & GRPC_PROPAGATE_CENSUS_TRACING_CONTEXT) {
+    char buf[GRPC_CENSUS_MAX_ON_THE_WIRE_TAG_BYTES];
+    size_t len = census_context_serialize(
```

Comment:
We should talk here... But there's no way we can take a link dependency on census from call.c.